modules/extension/app-schema/app-schema/src/main/java/org/geotools/data/complex/AppSchemaDataAccess.java

```diff
@@ -89,20 +92,33 @@
 
     private FilterFactory2 filterFac = CommonFactoryFinder.getFilterFactory2(null);
 
+    boolean allowCounts;
+
     /**
      * Flag to mark non-accessible data accesses, which should be automatically disposed of when no
      * longer needed by any accessible data access.
      */
     boolean hidden = false;
 
+    /**
+     * Constructor.
+     *
+     * @param mappings a Set containing a {@linkplain FeatureTypeMapping} for each FeatureType this
+     *     DataAccess is going to produce.
+     */
+    public AppSchemaDataAccess(Set<FeatureTypeMapping> mappings, boolean allowCounts)
```

Comment:
Update JavaDocs with the allowCounts parameter definition.

Reply:
fixed